src/main/java/com/PhamKornbluhGroup/SecretsHelper.java

```diff
@@ -0,0 +1,81 @@
+package com.PhamKornbluhGroup;
+
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueRequest;
+import software.amazon.awssdk.services.secretsmanager.model.GetSecretValueResponse;
+
+public class SecretsHelper {
+    // If you need more information about configurations or implementing the sample
+    // code, visit the AWS docs:
+    // https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/home.html
+
+    public static String[] getFormattedGGGBearerToken() {
+        String tokenName = "GGGBearerToken";
+        String secret = getSecretManagerSecret(tokenName);
+        String formattedSecret = formatGGGBearerToken(secret);
+
+        String[] keyValue = new String[] {"Authorization", formattedSecret};
+        return keyValue;
+    }
+
+    public static String[] getFormattedGGGBearerTokenUserAgent() {
+        String tokenName = "GGGBearerToken_User-Agent";
+        String secret = getSecretManagerSecret(tokenName);
+        String formattedSecret = formatGGGBearerTokenUserAgent(secret);
+
+        String[] keyValue = new String[] {"User-Agent", formattedSecret};
+        return keyValue;
+    }
+
+    private static String getSecretManagerSecret(String tokenName) {
+        Region region = Region.of("us-east-1");
+
+        //b Create a token request object
+        GetSecretValueRequest tokenRequest =
+                GetSecretValueRequest.builder()
+                                     .secretId(tokenName)
+                                     .build();
+
+        //b Create a client which can send the token request to AWS.
+        // Try-With-Resources Auto-Closes the client
+        try (SecretsManagerClient requestClient =
+                     SecretsManagerClient.builder()
+                                         .region(region)
+                                         // TODO: Determine if DefaultCredentialsProvider must be specified #2
+                                         //.credentialsProvider(DefaultCredentialsProvider.create())
+                                         .build()) {
+
+            //b Send the token request to AWS through the client
```

Comment:
Consider removing comments that don't explain why code behaves a certain way, or comments that don't points out an important nuance.

Reply:
Removed unneeded comment. Thanks!